package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
@guybedford Yes.  IIRC, I tried logging `imports[1]` to see what it was, and it was ultimately `{ default: etc }`.